kratos/processes/fast_transfer_between_model_parts_process.cpp

```diff
@@ -275,14 +304,25 @@ void FastTransferBetweenModelPartsProcess::ReplicateWithoutFlags()
 
         if (num_constraints != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::CONSTRAINTS || mEntity == EntityTransfered::NODESANDCONSTRAINTS)) {
             const auto it_const_begin = mrOriginModelPart.MasterSlaveConstraintsBegin();
-            #pragma omp for
+            #pragma omp for schedule(guided, 512)
             for(int i = 0; i < num_constraints; ++i) {
                 auto it_const = it_const_begin + i;
                 MasterSlaveConstraint::Pointer p_new_const = it_const->Clone(total_num_constraints + i + 1);
                 constraints_buffer_vector.insert(constraints_buffer_vector.begin(), p_new_const);
             }
         }
 
+        if (num_geometries != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::GEOMETRIES || mEntity == EntityTransfered::NODESANDGEOMETRIES)) {
+            const auto it_geom_begin = mrOriginModelPart.GeometriesBegin();
+            #pragma omp for
```

Comment:
Parallel utils.

Reply:
See the code as a whole, you cannot, there is a parallel area with criticals later